app/controllers/choices_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ChoicesController < ApplicationController
+
+  def create
+    @question = Question.find(params[:question])
+    text = params[:text]
+    correct = params[:correct] == '1'
```

Comment:
if 'correct' is always '1'. then why need to read from params ?

Reply:
I am not assigning here, just checking params[:correct] is 1 or not. that check gives us a boolean

Reply:
ohh didn't see '==', you can do samle thing by using  params[:correct].eul?('1')
http://www.rian.me/2013/10/15/what-is-the-difference-between-equals-equals-equals-and-equals-equals-in-ruby/